hydrolib/core/dflowfm/research/models.py

```diff
@@ -887,6 +888,45 @@ class Comments(Processes.Comments):
     )
 
 
+class ResearchSedTrails(INIBasedModel):
```

Comment:
Qua capitals, misschien is `ResearchSedtrails` logischer?

Reply:
Ik dacht dat het stond voor SedTRAILS, dus SedTrails leek my logisch. Maar als je de voorkeur hebt voor Sedtrails wil ik dat best aanpassen! Laat maar weten als je wil dat ik het aanpas!

Reply:
Dat weet ik niet precies, maar ik denk dat dit meer consequent is met de rest en ook met hoe Bert de onderliggende keywords noemt. Dat begint allemaal met `Sedtrails`, niet met `SedTrails`.

Reply:
Ik heb het aangepast naar `Sedtrails`